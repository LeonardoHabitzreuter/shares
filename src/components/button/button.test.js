import React from 'react'
import Button from './button'
import { shallow } from 'enzyme'

describe('button', () => {
  test('snapshot', () => {
    const button = shallow(<Button>My button</Button>)
    expect(button).toMatchSnapshot()
  })
})
