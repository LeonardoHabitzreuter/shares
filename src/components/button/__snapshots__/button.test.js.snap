// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button snapshot 1`] = `
<button
  className="btn btn-primary rounded"
>
  My button
</button>
`;
